socorro/collector/wsgi_breakpad_collector.py

```diff
@@ -86,6 +91,33 @@ def POST(self, *args):
         else:
             raw_crash.legacy_processing = int(raw_crash.legacy_processing)
 
+        # We want to capture the crash report size, but need to differentiate
+        # between compressed vs. uncompressed data as well as accepted vs.
+        # rejected data.
+        crash_report_size = self._get_content_length()
+        is_compressed = self._is_content_gzipped()
+        is_accepted = (raw_crash.legacy_processing in (ACCEPT, DEFER))
```

Comment:
Does this go against the socorro style guide? I haven't seen evidence of that, yet.

I like this style because it cordons off comparison expressions and makes it clearer that it's an assignment and not a typo of `==`. So I do this often.